examples/manage-dns/init.pp

```diff
@@ -0,0 +1,28 @@
+route53_zone { 'puppetlabs.com.':
+  ensure => present,
+}
+
+route53_a_record { 'local.puppetlabs.com.':
+  ensure => present,
+  zone   => 'puppetlabs.com.',
+  ttl    => 3000,
+  values => ['127.0.0.1'],
+}
+
+route53_ns_record { 'local.puppetlabs.com.':
+  ensure => present,
+  zone   => 'puppetlabs.com.',
+  ttl    => 17200,
+  values => [
+    'ns-377.awsdns-47.com.',
+    'ns-1724.awsdns-23.co.uk.',
+    'ns-969.awsdns-57.net.',
+  ]
+}
+
+route53_txt_record { 'local.puppetlabs.com.':
+  ensure => present,
+  zone   => 'puppetlabs.com.',
+  ttl    => 17200,
+  values => '"message"'
```

Comment:
The reasoning for why this is `'"message"'` instead of `'message'` is unclear, and the error message when not including the double quotes is not that helpful:

```
Error: /Stage[main]/Main/Route53_txt_record[local.test.plabs.net.]: Could not evaluate: Invalid Resource Record: FATAL problem: InvalidTXTRDATA encountered at message
```

Maybe some extra validation or munging would be helpful here?